ru/basecall.py

```diff
@@ -4,51 +4,35 @@
 
 """
 import logging
+import time
+from collections import namedtuple
 
 import mappy as mp
 import numpy as np
 
-from pyguppyclient.client import GuppyBasecallerClient
-from pyguppyclient.decode import ReadData as GuppyRead
-
+from pyguppy_client_lib.pyclient import PyGuppyClient
+from pyguppy_client_lib.helper_functions import package_read
 
 __all__ = ["GuppyCaller"]
 
 logger = logging.getLogger("RU_basecaller")
+CALIBRATION = namedtuple("calibration", "scaling offset")
+
+
+class DAQValues:
```

Comment:
Add in another DocString sentence about what the DAQValues does. i.e Returns the same default calibration values for any given key

Reply:
resolved in efe3f56